src/providers/eveonline.ts

```diff
@@ -0,0 +1,37 @@
+import { Profile } from ".."
+import { OAuthConfig, OAuthUserConfig } from "./oauth"
+
+export interface EVEOnlineProfile extends Profile {
+  CharacterID: number;
+  CharacterName: string;
+  ExpiresOn: string;
+  Scopes: string;
+  TokenType: string;
+  CharacterOwnerHash: string;
+  IntellectualProperty: string;
+}
+
+export default function EVEOnline<P extends Record<string, any> = EVEOnlineProfile>(
+  options: OAuthUserConfig<P>
+): OAuthConfig<P> {
+  return {
+    id: "eveonline",
+    name: "EVE Online",
+    type: "oauth",
+    token: "https://login.eveonline.com/oauth/token",
+    userinfo: "https://login.eveonline.com/oauth/verify",
+    authorization: {
+      params: { scope: "publicData" },
+      url: "https://login.eveonline.com/oauth/authorize",
+    },
+    profile(profile: P) {
+      return {
+        id: profile.CharacterID,
+        name: profile.CharacterName,
+        email: null,
```

Comment:
Any ideas if we could get the email also by default? maybe adding an `email` scope? Could you test it out? :pray: